docs/DeveloperGuide.adoc

```diff
@@ -721,17 +721,51 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 [width="59%",cols="22%,<23%,<25%,<30%",options="header",]
 |=======================================================================
 |Priority |As a ... |I want to ... |So that I can...
-|`* * *` |new user |see usage instructions |refer to instructions when I forget how to use the App
+|`* * *` | new user | see usage instructions | refer to instructions when I forget how to use the App
+|`* * *` | user | add a new person |
+|`* * *` | user | delete a person | remove entries that I no longer need
+|`* * *` | user | find a person by name | locate details of persons without having to go through the entire list
+|`* * *` | user | edit a person | change details of a person easily
+|`* * *` | user | tag a person | store some notes about certain people
+|`* * *` | user | categorize persons | easily organize contacts and later on find wrt group
+|`* * *` | user with many persons in the address book | find all the persons that satisfy part of the name (eg. Jo*) | search without remembering all the details
+|`* * *` | user | get closest result if my input doesn't match any name | I get results even if I misspell
+|`* * *` | user | add a picture to a person | recall easily who the person is
+|`* * *` | user | add a social media link to a person | get more info about a person in one place (in the App)
+|`* * *` | user | be warned before execution of a hazardous command | prevent from disastrous mistakes
+|`* * *` | user | change the theme of the GUI (dark/bright) | prevent eye strain or simply customize
+|`* * *` | user | validate persons details when adding |  prevent any mistakes
+|`* * *` | user | input more than 1 phone number | can store home + office + mobile numbers easily
+|`* * *` | user | search for people via different fields (address, phone number etc.) | easily view people of same area of residence
+|`* * *` | user | allow mass deletion | easy cleanup
+|`* * *` | user | have a favourites tag that is always easily accessible | easily access most frequent contacts
+|`* * *` | forgetful user | show my phone number | so that I can give it to my friend when they ask
+|`* *` | user with many persons in the address book | apply filter: category, tag, name | easily view and process only certain group
+|`* *` | user | redo the command | correct the mistake of undoing too many commands
+|`* *` | user | view the history of commands | check what I have done in the past
+|`* *` | user | hide private contact details by default | minimize chance of someone else seeing them by accident
```

Comment:
I realize that this is actually similar to the hidden contact below. Maybe remove one and standardize on the naming?